<?xml version="1.0"?>

<launch>
	<rosparam command="load" 		    file="$(find cfswarm)/config/namespaces.yaml" />
	<rosparam command="load" 		    file="$(find adaptive)/config/limits.yaml" />	
    <rosparam command="load" 		    file="$(find cfswarm)/config/D3SmallClusterPoints4x3.yaml" />

    <param name="gamma"			    value="0.01" />
    <param name="threshold"			    value="2" /> <!-- comms range in meters-->
    <param name="avoidRadius"			value="0.4" />  <!-- avoidance radius Original value:0.7-->
    
    <param name="rho"					value="0.0" />  <!-- CRW  parameter 	-->
	<param name="xvel"					value="0.2" />  <!-- forward vel 	 	-->
	<param name="xvelMax"				value="0.2" />
	<param name="xvelMin"				value="0.05" />
	<param name="Kw"					value="100"/> <!-- Angular turning gain -->
	<param name="sensingRadius"		value="0.1" /> 
	<param name="scale"					value="10" /> 
	<param name="height"				value="1.0" />  <!-- takeoff height 	 	-->
	<param name="usevicon"				value="false" /> 

	<param name="mu"					value="2" />
    <param name="priority"				value="firefly" />
	<param name="decay"				value="linear" />
        
    <include file="$(find cfswarm)/launch/cfswarm.launch" />

    <node pkg="cfswarm"    	type="firefly" 						name="firefly" 						output="screen"/> 
    <node pkg="cfswarm"   	type="getAverageMapServer.py" 		name="getAverageMapServer" 		output="screen"/> 
	
 
</launch>
